packages/dx-react-scheduler/src/plugins/appointments.test.jsx

```diff
@@ -101,6 +106,8 @@ describe('Appointments', () => {
     expect(appointmentContentData.title).toBe('a');
     expect(appointmentContentData.endDate).toBe('2018-07-05');
     expect(appointmentContentData.startDate).toBe('2018-07-06');
+
```

Comment:
It will be more readable to compare objects with fields { title, endData, startDate  }